### DISCUSSION TITLE
Unable to create roboshop instances

### DISCUSSION THREAD

**Discussion by Yaalinilakshmi** [learndevopsonline/learndevopsonline#1474] [Q&A]
Hi All,

I am trying to create roboshop instances from AWS cli but it is throwing out a message that given security group does not exist.Please do let me know what is the security  group that it is expecting?? below is  the screenshot

![Uploading roboshop instances.PNG…]()

**Comment by sreedharm07** [comment]
your screen shot is not able to view can you please send screen shot again
